public/js/utils/debug.js

```diff
@@ -1,11 +1,10 @@
 const { isDevelopment } = require("../feature");
+const flag = require("../test/test-flag");
```

Comment:
> see webpack.config.devtools.js changes

ahh, I see what you mean. 

I think you could also add an `isTesting` method in `feature` alongside `isDevelopment`. 

I trust your judgement though.